pkg/command/permission_service_test.go

```diff
@@ -0,0 +1,223 @@
+package command_test
+
+import (
+	is "github.com/arquillian/ike-prow-plugins/pkg/command"
+	"github.com/arquillian/ike-prow-plugins/pkg/github"
+	. "github.com/arquillian/ike-prow-plugins/pkg/internal/test"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/h2non/gock.v1"
+)
+
+var _ = Describe("Permission service with permission checks features", func() {
+
+	Context("Getting and validating user's permissions", func() {
+
+		client := NewDefaultGitHubClient()
+		var user is.PermissionService
+
+		BeforeEach(func() {
+			gock.Off()
+			user = is.PermissionService{
+				Client: client,
+				User:   "user",
+				PRLoader: &github.PullRequestLazyLoader{
+					Client:    client,
+					RepoOwner: "owner",
+					RepoName:  "repo",
+					Number:    1,
+				},
+			}
+		})
+
+		It("tests 'that is admin' function that should not approve the user when the permission is read", func() {
```

Comment:
Do we really need `tests 'that is admin' function that ` prefixes in those tests?

Reply:
it seems that it doesn't have to be there :-D. To be honest, I have no idea why I added it there. The reason was that when I combined the text with the name of the function `It`, then it didn't sound natural to me:
`it - should not approve the user when the permission is read"`
but I have no idea why :-D. 
I'll remove it ;-)

Reply:
removed in 0c91c60